tox.ini

```diff
@@ -74,7 +74,7 @@ commands =
     black --target-version=py310 --check --diff {[vars]all_path}
     mypy {[vars]all_path}
     pydocstyle {[vars]src_path}
-    pylint {[vars]all_path}
+    pylint {[vars]all_path} --min-similarity-lines=10
```

Comment:
Out of curiosity, may I ask where the similarities were?

Reply:
Some classes take multiple arguments in their constructors, and black will split the function call across several lines. This splitting process results in the failure of the similarity check.

Reply:
Thanks for the explanation